### ISSUE TITLE
Any chance to upgrade this library to use ldapjsv3?

### ISSUE THREAD

**Issue by tfrancois** [txstate-etc/ldap-async#10]
I was wondering if there was any chance that this excellent library would be upgraded to take advantage of the changes in ldapjs v3 - which is a major update to v2?  Thanks.

**Comment by wickning1** [comment]
Yes, I’ll take a look. It seems like they have a new Filters API that I haven’t seen before, looks pretty nice.

**Comment by genio** [comment]
Looks like this can be closed

**Comment by wickning1** [comment]
I'm working on this this week. They really changed the API in 3 so I am changing my API too and will cut a new major version.

**Comment by tfrancois** [comment]
Fantastic news!  Thank you good sir - please keep us posted.

**Comment by wickning1** [comment]
I coded everything up and put it up as PR #12 for a few days before release.

**Comment by wickning1** [comment]
Just as an update, we discovered an issue with ldapjs 3 where it will not support range requests on attributes, so this is blocked pending a successful PR. We're working on that PR now.

**Comment by wickning1** [comment]
ldap-async 2.0 is published

**Comment by tfrancois** [comment]
@wickning1 You're a rock star! Installing today - thank you sir!
